### PR TITLE
feat: badge

### PR DESCRIPTION
**Pull request by shivangisharma01**
image

image

### CODE REVIEW

File: components/alert/Badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+.badge .away {
+    background-color: var(--away-color);
+
+}
+
+.badge .badge-counts {
+    display: flex;
+    justify-content: center;
+    align-items: center;
```
Comment (nakulsharma15) [inline components/alert/Badge/badge.css:48]: Just a suggestion. You can extract this in a utility class

**Review by nakulsharma15 — APPROVED** [review]
Gave a small suggestion. Everything else looks good

**Review by sadathshariff — APPROVED** [review]
Everything looks good

File: components/alert/Badge/badge.css (hunk 1 above)
Comment (shivangisharma01) [reply components/alert/Badge/badge.css:48]: okay.

**Pull request merged**